beacon-chain/rpc/eth/config/handlers_test.go

```diff
@@ -170,293 +170,309 @@ func TestGetSpec(t *testing.T) {
-			assert.Equal(t, "0x0a000000", v)
-		case "DOMAIN_APPLICATION_BUILDER":
-			assert.Equal(t, "0x00000001", v)
-		case "DOMAIN_BLOB_SIDECAR":
-			assert.Equal(t, "0x00000000", v)
-		case "TRANSITION_TOTAL_DIFFICULTY":
-			assert.Equal(t, "0", v)
-		case "TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH":
-			assert.Equal(t, "72", v)
-		case "TERMINAL_BLOCK_HASH":
-			s, ok := v.(string)
-			require.Equal(t, true, ok)
-			assert.Equal(t, common.HexToHash("TerminalBlockHash"), common.HexToHash(s))
-		case "TERMINAL_TOTAL_DIFFICULTY":
-			assert.Equal(t, "73", v)
-		case "DefaultFeeRecipient":
-			assert.Equal(t, common.HexToAddress("DefaultFeeRecipient"), v)
-		case "PROPORTIONAL_SLASHING_MULTIPLIER_BELLATRIX":
-			assert.Equal(t, "3", v)
-		case "MIN_SLASHING_PENALTY_QUOTIENT_BELLATRIX":
-			assert.Equal(t, "32", v)
-		case "INACTIVITY_PENALTY_QUOTIENT_BELLATRIX":
-			assert.Equal(t, "16777216", v)
-		case "PROPOSER_SCORE_BOOST":
-			assert.Equal(t, "40", v)
-		case "INTERVALS_PER_SLOT":
-			assert.Equal(t, "3", v)
-		case "MAX_WITHDRAWALS_PER_PAYLOAD":
-			assert.Equal(t, "74", v)
-		case "MAX_BLS_TO_EXECUTION_CHANGES":
-			assert.Equal(t, "75", v)
-		case "MAX_VALIDATORS_PER_WITHDRAWALS_SWEEP":
-			assert.Equal(t, "76", v)
-		case "REORG_MAX_EPOCHS_SINCE_FINALIZATION":
-			assert.Equal(t, "2", v)
-		case "REORG_WEIGHT_THRESHOLD":
-			assert.Equal(t, "20", v)
-		case "REORG_PARENT_WEIGHT_THRESHOLD":
-			assert.Equal(t, "160", v)
-		case "MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT":
-			assert.Equal(t, "8", v)
-		case "MAX_REQUEST_LIGHT_CLIENT_UPDATES":
-			assert.Equal(t, "128", v)
-		case "SAFE_SLOTS_TO_IMPORT_OPTIMISTICALLY":
-		case "NODE_ID_BITS":
-			assert.Equal(t, "256", v)
-		case "ATTESTATION_SUBNET_EXTRA_BITS":
-			assert.Equal(t, "0", v)
-		case "ATTESTATION_SUBNET_PREFIX_BITS":
-			assert.Equal(t, "6", v)
-		case "SUBNETS_PER_NODE":
-			assert.Equal(t, "2", v)
-		case "EPOCHS_PER_SUBNET_SUBSCRIPTION":
-			assert.Equal(t, "256", v)
-		case "MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS":
-			assert.Equal(t, "4096", v)
-		case "MAX_REQUEST_BLOB_SIDECARS":
-			assert.Equal(t, "768", v)
-		case "MESSAGE_DOMAIN_INVALID_SNAPPY":
-			assert.Equal(t, "0x00000000", v)
-		case "MESSAGE_DOMAIN_VALID_SNAPPY":
-			assert.Equal(t, "0x01000000", v)
-		case "ATTESTATION_PROPAGATION_SLOT_RANGE":
-			assert.Equal(t, "32", v)
-		case "RESP_TIMEOUT":
-			assert.Equal(t, "10", v)
-		case "TTFB_TIMEOUT":
-			assert.Equal(t, "5", v)
-		case "MIN_EPOCHS_FOR_BLOCK_REQUESTS":
-			assert.Equal(t, "33024", v)
-		case "GOSSIP_MAX_SIZE":
-			assert.Equal(t, "10485760", v)
-		case "MAX_CHUNK_SIZE":
-			assert.Equal(t, "10485760", v)
-		case "ATTESTATION_SUBNET_COUNT":
-			assert.Equal(t, "64", v)
-		case "MAXIMUM_GOSSIP_CLOCK_DISPARITY":
-			assert.Equal(t, "500", v)
-		case "MAX_REQUEST_BLOCKS":
-			assert.Equal(t, "1024", v)
-		case "MAX_REQUEST_BLOCKS_DENEB":
-			assert.Equal(t, "128", v)
-		default:
-			t.Errorf("Incorrect key: %s", k)
-		}
+		t.Run(k, func(t *testing.T) {
```

Comment:
Formatting here to have subtests within the tests to explain which key was failing.